src/components/widgets/ModalCategories.svelte

```diff
@@ -1,27 +1,25 @@
+
 <script>
-	import InputButton from '../blocks/InputButton.svelte';
-	import Modal from '../blocks/Modal.svelte';
-	import ScrollableList from '../blocks/ScrollableList.svelte';
-	import store from '../../utils/store';
-	import { close, name } from '../../models/modalManager';
+  import InputButton from '../blocks/InputButton.svelte';
+  import Modal from '../blocks/Modal.svelte';
+  import ScrollableList from '../blocks/ScrollableList.svelte';
+  import store from '../../utils/store';
+  import { close, name } from '../../models/modalManager';
+  import { Category } from '../../models/categories';
 
-	function mapCategories(name) {
-	    return { name, old: name, disabled: true};
-	}
-	let categories = store.getState().categories.map(mapCategories);
-	store.subscribe(() => categories = store.getState().categories.map(mapCategories));
+  let newCategory = new Category();
```

Comment:
I prefer `const` in this case
`const newCategory = new Category();`

Reply:
Theres some problem with reactivity in svelte, view don't see changes in newCategory. so i need to do stuff like `newCategory = newCategory`. And this stuff will not work with `const`

Sad but true